models/intermediate/int_recharge__customer_daily_rollup.sql

```diff
@@ -13,7 +13,7 @@ with calendar as (
         cast({{ dbt.date_trunc('week', 'calendar.date_day') }} as date) as date_week,
         cast({{ dbt.date_trunc('month', 'calendar.date_day') }} as date) as date_month,
         cast({{ dbt.date_trunc('year', 'calendar.date_day') }} as date) as date_year
-    from calendar, customers
+    from calendar, customers 
```

Comment:
Can we do an explicit cross join here? Just to make it more readable for customers checking out the code.